packages/frontend/src/components/campaigns/campaign/index.tsx

```diff
@@ -39,26 +39,26 @@ export function Campaign({ campaign }: CampaignProps) {
                     status={campaign.status}
                 />
                 <Apr apr={campaign.apr} kpi={!!campaign.specification?.kpi} />
-                {campaign.points && (
+                {campaign.distributables.type === "points" && (
```

Comment:
I think we can define a `const` for this, same for line 49, since we use them multiple times.

Reply:
I was also thinking about making the campaigns stuff an object with dynamically calculated properties (such as `Status`) and functions such as `distributesPoints()` or `distributesTokens()`, honestly. Wdyt?

Reply:
Yeah, that probably would tidy things up a bit.

Reply:
Addressed in 544ffd7fbd20a18ca23188669caf62ed148ace8c and cb6025980f5de99983976b083a08b88bf6548d02